api/src/main/java/com/community/api/endpoint/avisoft/controller/Account/AccountEndPoint.java

```diff
@@ -404,7 +404,11 @@ else if(mobileNumber!=null)
                     customAdmin=adminService.findAdminByPhone(mobileNumber,countryCode);
                 }
                 if (customAdmin == null) {
-                    return responseService.generateErrorResponse("Custom Admin "+" not found", HttpStatus.NOT_FOUND);
+                    return responseService.generateErrorResponse("Custom Admin with "+ usernameOrMobileNumber+" not found", HttpStatus.NOT_FOUND);
+                }
+                if(!customAdmin.getPassword().equals(password))
+                {
+                    return ResponseService.generateErrorResponse("You have entered wrong password",HttpStatus.BAD_REQUEST);
                 }
                 if (customAdmin.getRole() ==2) {
```

Comment:
should check role from contstants like :   if (customAdmin.getRole().equals(Constant.roleAdmin) {